ci/templates/install_deps_caliper.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+steps:
+  - script: sudo apt-get clean
+    displayName: Run APT Clean
+
+  - script: sudo apt-get update
+    displayName: Run APT Update
+
+  - script: sudo apt-get install -y gcc haveged libtool make
+    displayName: Install Dependencies
+
+  - task: GoTool@0
+    inputs:
+      goBin: $(GO_BIN)
+      goPath:  $(GO_PATH)
+      version: $(GO_VER)
+    displayName: Install GoLang
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: 10.x
```

Comment:
Can we use 12.x since all other pipelines use 12.x?

Reply:
Unfortunately we cant, 12.15 breaks the pipeline (they only support 8.x and 10.x, but prefer 8.x) but 10.x does work

Reply:
ok then I am good with the changes.